src/Domain/BoundaryConditions/GetBoundaryConditionsBase.hpp

```diff
@@ -0,0 +1,34 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+#include "Utilities/TypeTraits/CreateHasTypeAlias.hpp"
+
+namespace domain::BoundaryConditions {
+CREATE_HAS_TYPE_ALIAS(boundary_conditions_base)
+CREATE_HAS_TYPE_ALIAS_V(boundary_conditions_base)
```

Comment:
As we transition from the current method of handling boundary conditions to the new, it would be good not to break develop for a month or two, I think. These are used to allow switching one system over at a time.